ColorPalette.js

```diff
@@ -5,12 +5,11 @@ define([
 	"./_TemplatedMixin",
 	"./_PaletteMixin",
 	"./hccss", // has("highcontrast")
-	"dojo/i18n", // i18n.getLocalization
```

Comment:
Oh that's a good idea.   I like the idea of doing it whenever we are using the async loader.  Can you branch based on that `async` flag that we usually set when we include dojo.js?

```
<script type="text/javascript" src="dojo.js"
	data-dojo-config="async: true"></script>
```

Hopefully that's exposed as a has() flag.